src/flash-source-buffer.js

```diff
@@ -182,11 +182,18 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
       removeCuesFromTrack(0, Infinity, this.inbandTextTrack_);
     });
 
+    this.mediaSource_.player_.tech_.on('hls-reset', this.resetHls);
```

Comment:
Looks like `bind` is being used in a couple other places there, maybe it's best to do that to keep the style consistent.